store/tikv/region_cache.go

```diff
@@ -171,11 +171,31 @@ func (r *RegionStore) kvPeer(seed uint32) AccessIndex {
 		}
 		candidates = append(candidates, AccessIndex(i))
 	}
-
+	candidates = r.filterStoreCandidates(op, candidates...)
```

Comment:
Why not call `r. filterStoreCandidate` in the loop above? Thus you can remove the function `filterStoreCandidates`.

Reply:
updated.